packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -123,14 +139,24 @@ export class ChatWidget extends Component {
 
   handleScroll() {
     const props = this.props;
+    const {
+      conversation,
+      spark,
+      widget
+    } = props;
+
+    props.setScrollPosition({scrollTop: this.activityList.getScrollTop()});
```

Comment:
Ahhh got it. Can't do that in this case because setScrollPosition is imported at the top level scope.